lib/rouge/lexers/docker.rb

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*- #
+
+module Rouge
+  module Lexers
+    class Docker < RegexLexer
+      title "Docker"
+      desc "Dockerfile syntax"
+      tag 'docker'
+      aliases 'docker', 'dockerfile'
+      filenames 'Dockerfile', '*.docker'
+      mimetypes 'text/x-dockerfile-config'
+
+      def self.analyze_text(text)
+        return 0.8 if text =~ /\AFROM\s+.+/im
```

Comment:
I used `/i` switch because Dockerfile instructions are not case-sensitive: http://docs.docker.com/engine/reference/builder/#format
But I didn't know about values above `0.5`. Will fix it soon.